src/test/functional/features/form-failures.feature

```diff
@@ -401,3 +401,45 @@ Feature: Form failures
     When I click "Submit"
     Then the page should include "You need to confirm you are applying to the court to dissolve your marriage (get a divorce)."
     And the page should include "You need to confirm the facts stated in this application are true"
+
+    @nightly
+    Scenario: They fail to fill out joint conditional order questions
+      Given I've already completed the form using the fixture "jointApplicant1CompleteCase"
+      When I go to "/"
+      And I click "Send for review"
+      Then the page URL should be "/application-sent-for-review"
+      When I enter my valid case reference and valid access code
```

Comment:
is it possible to have an empty line between the different steps, just so that it's easier to see?

Reply:
same with the above, I'll create a separate pr to do this since there are quite a few e2e tests that could be split up in this way